policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/retirement_income_exemption/vt_military_retirement_income_based_exemption.py

```diff
@@ -0,0 +1,54 @@
+from policyengine_us.model_api import *
+
+
+class vt_military_retirement_income_based_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Vermont military retirement income-based exemption"
+    reference = "https://legislature.vermont.gov/Documents/2026/Docs/BILLS/S-0051/S-0051%20As%20Passed%20by%20Both%20House%20and%20Senate%20Official.pdf#page=10"
+    unit = USD
+    defined_for = StateCode.VT
+    documentation = "Vermont military retirement benefits exempt from Vermont taxation based on AGI thresholds (2025+)."
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.vt.tax.income.agi.retirement_income_exemption.military_retirement
+
+        # Get retirement amount from military retirement system
+        tax_unit_military_retirement_pay = add(
+            tax_unit, period, ["military_retirement_pay"]
+        )
+
+        agi = tax_unit("adjusted_gross_income", period)
+
+        # Full exemption for households under $125k AGI
```

Comment:
Please dont use specific amounts in the variable comments. Once the values change, our variable and parameter comment structure won't align.